src/main/java/com/google/codeu/servlets/ChartsDataServlet.java

```diff
@@ -81,17 +83,18 @@ else if(analysisType.equals("lastSevenDays")) {
 	private void getLastSevenDays(HttpServletRequest request, HttpServletResponse response) throws IOException {
 		response.setContentType("application/json");
 		String user = request.getParameter("user");
-		//Get the time seven days ago
-		LocalDate date = LocalDate.now().minusDays(7);
-		//Convert it to old style Date object
-		Date sevenDaysAgo = (Date) java.sql.Date.valueOf(date);
+		
+		//Get date from 7 days ago
+		Instant now = Instant.now();
+		Instant before = now.minus(Duration.ofDays(7));
+		Date sevenDaysAgo = Date.from(before);
 		
 		FilterPredicate dateFilter = new Query.FilterPredicate("date", Query.FilterOperator.GREATER_THAN, sevenDaysAgo);
 		FilterPredicate userFilter = new Query.FilterPredicate("user", Query.FilterOperator.EQUAL, user);
 		
 		Query query = new Query("UserMeal")
```

Comment:
I think it's called EatenMeal right now

Reply:
i think all these changes are because my pr isn't merged rn so can be ignored for the moment.